lib/run-status.js

```diff
@@ -123,6 +125,10 @@ RunStatus.prototype.handleTest = function (test) {
 		this.errors.push(test);
 	}
 
+	if (test.failing && !test.error) {
+		this.knownFailures.push(test);
```

Comment:
where should I write tests for this line? Seems that there is no test/run-status.js